harmonic/ros2_launch_gazebo.md

```diff
@@ -104,18 +104,26 @@ def generate_launch_description():
 Here's another example using a higher level action from Python to launch `gzserver`:
 ```python
 from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration, TextSubstitution
 from ros_gz_sim.actions import GzServer
 
 
 def generate_launch_description():
 
+    declare_world_sdf_file_cmd = DeclareLaunchArgument(
+        'world_sdf_file', default_value=TextSubstitution(text=''),
```

Comment:
Is the `TextSubstitution` necessary?

```suggestion
        'world_sdf_file', default_value=''",
```

Reply:
Removed in [24a8cf3](https://github.com/gazebosim/docs/pull/509/commits/24a8cf3d8af0043b810acfafd9588abfb8d0bf1d).